new-dti-website/src/app/course/page.tsx

```diff
@@ -115,19 +115,19 @@ export default function Courses() {
               </div>
 
               <div className="flex flex-col lg:w-1/2">
-                <div className="font-black text-sm md:text-xl tracking-wider">
+                <div className="font-semibold text-sm md:text-xl">
                   MODERN INDUSTRY-LEADING TECHNOLOGY
                 </div>
 
-                <div className="font-black text-4xl tracking-wider mt-4 md:text-[45px]">
+                <div className="font-bold text-4xl mt-4 md:text-[45px]">
                   Trends in Web Development
                 </div>
 
-                <div className="text-md md:text-2xl mt-8">
+                <div className="text-md md:text-xl mt-8">
```

Comment:
@JasonMun7 I believe we're using `md:text-lg` for the body text on every other page. I tried to standardize it but looks like I missed it here!

Also side note but I don't think `text-md` refers to a valid CSS class :P, maybe remove it?

Reply:
Yes of course! Also you're right about `text-md` not being valid haha should be `text-base`! I also changed it for the other body text within the page as well to be md:text-lg! Thanks Andrew!